src/Plugins/ApplicationLogs/ApplicationLogsPlugin.cs

```diff
@@ -26,63 +27,61 @@
 
 namespace Neo.Plugins.ApplicationLogs
 {
-    public class LogReader : Plugin, ICommittingHandler, ICommittedHandler, ILogHandler
```

Comment:
will this affect the function in CLI.Plugin? like when we list the plugins, can we still get the ApplicationLogs?

Reply:
No you will not see `ApplicationLogs` in the plugins. Future we will not have plugins. It will be features.

Reply:
This won't list the plugin by RPC